docs/source/specs/openapi.json

```diff
@@ -143,6 +143,16 @@
               ],
               "default": "false"
             }
+          },
+          {
+            "in": "query",
+            "name": "order_by",
```

Comment:
Overall LGTM! Unfortunately, /principals does not support "order_by" yet, we only have a sort_order which works for username only.

Reply:
Oh! That's odd, the /groups/uuid/principals/ claims to support it at line 675, should I pull that one out too or does it use a different mechanism?

Reply:
Right, they have different implementation, /principals/ only supports sorting by username currently, so there is no sort_by option, but sort_order. /groups/uuid/principals/ has order_by implemented without sort_order.